<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nantong_login_dialog</class>
 <widget class="QDialog" name="nantong_login_dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#nantong_login_dialog
{
	border: 1px solid black;
	background-color:white;
}

    QMessageBox
    {
        background-color:white;
        border: 1px solid black;
    }
    QPushButton
    {
        padding-left:35px;
        padding-right:35px;
        padding-top:5px;
        padding-bottom:5px;
        border: no border;
        background-color:#92CDDC
    }
    QPushButton:hover
    {
        background-color:#C4D79B;
    }


QToolButton
{
background-color: white;
border-width: 0px;
border-radius: 0;
}

QToolButton:hover
{
background-color: #92CDDC;
}

QPushButton:disabled {
background-color:#CCCCCC;
}

QLineEdit
{
background-color:white;
border: 1px solid black;
}

QLineEdit:hover
{
border: 1px solid red;
}</string>
  </property>
  <widget class="QLineEdit" name="edit_user">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>130</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_password">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>170</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_remember_pass">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_login">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>160</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;img src = &quot;nantongsanjian_logo.png&quot; width = &quot;160&quot; height = &quot;70&quot;&gt; &lt;/img&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>1</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edit_user</tabstop>
  <tabstop>edit_password</tabstop>
  <tabstop>button_login</tabstop>
  <tabstop>checkbox_remember_pass</tabstop>
  <tabstop>button_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>nantong_login_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_user</sender>
   <signal>returnPressed()</signal>
   <receiver>edit_password</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_password</sender>
   <signal>returnPressed()</signal>
   <receiver>button_login</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>nantong_login_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
